Custom Magento icon css and use in our theme (#4135)
magento/devdocs: guides/v2.1/frontend-dev-guide/css-topics/using-fonts.md

        font-style: normal
    }
    ```
## Overview of Magento's Icon CSS

In addition to including custom fonts in your Magento Blank theme, you also can include custom fonts for any icons in the Blank theme. The icon font files for the Magento Blank theme are located in the `lib/web/fonts/Blank-Theme-Icons` directory. The `lib/web/css/source/lib/variables/_typography.less` file defines the font icon path and name for the icons and the `web/css/source/_icons.less` file uses these files to define the icon font face itself, which should be used in all CSS declarations.
 
 The Unicode characters that correspond to the correct font glyphs for each icon are defined in the following `lib/web/css/source/lib/variables/_icons.less` file.

 By calling the `lib-icon-font` mixin, you can apply the icon font and character variables throughout the theme LESS code.

> 

Reviewer: Delete this extra return. The rest looks good to me.

Author: @eriksmarr I have completed suggested changes, please review

Thanks for the helping :)

        font-style: normal
    }
    ```
## Overview of Magento's Icon CSS

 To find the icon font files for the Magento Blank theme in `lib/web/fonts/Blank-Theme-Icons`

 

 The Unicode characters that correspond to the correct font glyphs for each icon are defined in the following `lib/web/css/source/lib/variables/_icons.less` file.

>  The defined icon font face and the character variables are used throughout the theme LESS code to define icons, usually through a call to the `lib-icon-font` mixins. And you’ve successfully replaced the native icons with your own. 

Reviewer: ```suggestion
 By calling the `lib-icon-font` mixin, you can apply the icon font and character variables throughout the theme LESS code.
```

>  The defined icon font face and the character variables are used throughout the theme LESS code to define icons, usually through a call to the `lib-icon-font` mixins. And you’ve successfully replaced the native icons with your own. 

Author: PR is about applying custom fonts to existing icons. With the same process, we can change the frontend theme native icons too.

>  The defined icon font face and the character variables are used throughout the theme LESS code to define icons, usually through a call to the `lib-icon-font` mixins. And you’ve successfully replaced the native icons with your own. 

Reviewer: Ok so just to clarify, you can also change the fonts on the frontend theme native icons?

> Using [IcoMoon](https://icomoon.io/app/), you can start with an imported font like Magento's native icons, make your desired tweaks.

Reviewer: ```suggestion
To customize a font you import, consider using [IcoMoon](https://icomoon.io/app/).
```
